### DISCUSSION TITLE
Precision parameter

### DISCUSSION THREAD

**Discussion by siebe-hiel** [open-edge-platform/anomalib#965] [Q&A] [answered]
For certain parameters, I cannot find the meaning of them. What is does the precision parameter do in the config file? Is there a readme for all the paremeters in the config.yaml files?

**Comment by samet-akcay** [comment, marked as answer]
Hi @siebe-hiel, as stated [here](https://github.com/openvinotoolkit/anomalib/blob/main/src/anomalib/models/padim/config.yaml#L68), those are PyTorch Lightning `Trainer` parameters. For more details, you could refer to the trainer [documentation](https://lightning.ai/docs/pytorch/stable/common/trainer.html#trainer-class-api). Let us know if this is any helpful.

**Reply by siebe-hiel** [reply]
This is super usefull, thanks a lot!
